datalad/interface/run.py

```diff
@@ -713,14 +713,13 @@ def run_command(cmd, dataset=None, inputs=None, outputs=None, expand=None,
     msg = msg.format(
         message if message is not None else _format_cmd_shorty(cmd),
         '"{}"'.format(record_id) if use_sidecar else record)
-    msg = assure_bytes(msg)
```

Comment:
what about introducing a dedicated test (or enabling some existing one in this code base which currently would fail due to this bug?) for appveyor to avoid a wild goose chase?

Reply:
Not sure what goose is being changed. I don't see what test could be written to make `run` work on windows without investing in a range of windows bug fixes first.

Let me know if you oppose such changes and I'll tackle this by an RF that allows me to replace this from outside.

Reply:
Please don't blackmail, I do not oppose the change, I just want to avoid a blind fix. A possible RF to do is to take this saving functionality and move into a dedicated method to be called here, and tested independently without requiring the whole run to work properly

Reply:
Did you see the failure log lonked in #3030? It fails due to a mixture of strong and byte args in the command invocation. I don't see how I could test that, or what the advantage would be if testing that the command would then fail for other reasons.